src/libLookup/Lookup.cpp

```diff
@@ -1519,8 +1515,7 @@ bool Lookup::ProcessSetDSBlockFromSeed(const vector<unsigned char>& message,
     LOG_EPOCH(INFO, to_string(m_mediator.m_currentEpochNum).c_str(),
               "I already have the block");
   } else {
-    if (m_syncType == SyncType::NO_SYNC &&
```

Comment:
Why changed this condition

Reply:
When the node sent the solution to ds, it has stopped mining.

Reply:
But this is ProcessSetDSBlockFromSeed, which is the response when the node calls GetDSBlockFromLookup, what's the relationship of sending the solution to ds here?

Reply:
New node sends ds pow solution -> waits ->checks if ds block has been mined

Reply:
I see, seems we no longer use this for synchronization. specifically used for checking pow